storage/cloud-client/README.rst

```diff
@@ -154,7 +154,7 @@ To run this sample:
     
```

Comment:
This file is generated, you'll need to update `README.rst.in` and run `nox -s "readmegen('./storage/cloud-client')"`